src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
-
+// not my solution
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averGrade'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order:SortOrder,
+): Student[] {
+  const sorted: Student[] = [...students];
```

Comment:
What is sorted? Rename this variable
```suggestion
  const sorted: Student[] = [...students];
```